README.md

```diff
@@ -424,12 +424,26 @@ Http source requires success codes defined in parameter: `gid.connector.http.sou
 which are considered as success response. It may be 200 (ok) as well as 404 (not found). The first one is standard response and its content should be deserialized/parsed.
 Processing of 404 request's content may be skipped by adding it to parameter `gid.connector.http.source.lookup.ignored-response-codes`.
 
-Both parameters supports whitelisting and blacklisting (ops!). Sample configuration may look like:
+Both parameters support whitelisting and blacklisting (oops!). A sample configuration may look like this:
 `2XX,404,!203` - meaning all codes from group 2XX (200-299), with 404 and without 203 ('!' character). Group blacklisting e.g. !2XX is not supported.
 Notice that ignored-response-codes has to be a subset of success-codes.
 
 The same format is used in parameter `gid.connector.http.source.lookup.retry-codes`.
 
+Example with explanation:
+```roomsql
+CREATE TABLE [...]
+WITH (
+  [...],
+  'gid.connector.http.source.lookup.success-codes' = '2XX,404',
+  'gid.connector.http.source.lookup.retry-codes' = '5XX,!501,!505,!506',
+  'gid.connector.http.source.lookup.ignored-response-codes' = '404'
+)
+```
+All 200s codes and 404 are considered as successful (`success-codes`). These responses won't cause retry or job failure. 404 response is also listed in `ignored-response-codes` parameter, what means content body will be ignored. Http with 404 code will produce just empty record. Notice that 404 has to be specified in both `success-codes` and `ignored-response-codes`.
+When server returns response with 500s code except 501, 505 and 506 then connector will re-send request based on configuration in `gid.connector.http.source.lookup.retry-strategy` parameters. By default it's fixed-delay with 1 second delay, up to 3 times per request (parameter `lookup.max-retries`).
```

Comment:
> By default it's fixed-delay with 1 second delay, up to 3 times per request
What happens when request fails 3 times? Do we return empty result or the job fails?